frontend/package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "build": "./node_modules/.bin/webpack --config webpack.config.js",
+    "prewatch": "npm install",
```

Comment:
А как этот скрипт вызывается? Автоматически? оО

Reply:
@vas3k 
`One of the nice features of npm is a pre and post hooks. For example if you execute npm run watch it will first run prewatch command, followed by watch, and postwatch.`

Reply:
Наделают неявных фич, а питонисты потом страдают!